travisci/python-linter_harness.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PYTHON_SOURCE_LOCATIONS=('scripts')
+
+PYLINT_OUTPUT_FILE=$(mktemp)
+PYLINT_ERRORS=$(mktemp)
```

Comment:
Can we rely on `mktemp` being available everywhere?

Reply:
Not in windows: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/mktemp?view=vs-2019

Reply:
On linux, yes it's present everywhere. If I try to remove `coreutils` (the package it's part of), it says
```
root@d48ff7c56ce2:/# apt-get remove coreutils
(...)
WARNING: The following essential packages will be removed.
This should NOT be done unless you know exactly what you are doing!
  coreutils
(...)
You are about to do something potentially harmful.
To continue type in the phrase 'Yes, do as I say!'
 ?```